WcaOnRails/spec/models/competition_spec.rb

```diff
@@ -598,4 +598,15 @@
       expect(competition.competitors.count).to eq 2
     end
   end
+
+  context "searching with #contains" do
```

Comment:
What I thought is
`describe "#contains?"`

- we are describing/testing the method so `describe` fits here better than `context`
- when testing a single method it's good to use its name as the `describe` description
- by convention `#` prefix goes for instsnce methods and `.` for class ones